bftengine/src/bftengine/ReplicaImp.hpp

```diff
@@ -104,8 +104,8 @@ class ReplicaImp : public InternalReplicaApi, public ReplicaForStateTransfer {
   SeqNum maxSeqNumTransferredFromPrevViews = 0;
 
   // requests queue (used by the primary)
-  std::queue<ClientRequestMsg*> requestsQueueOfPrimary;  // only used by the primary
-  size_t primaryCombinedReqSize = 0;                     // only used by the primary
+  std::list<ClientRequestMsg*> requestsQueueOfPrimary;  // only used by the primary
```

Comment:
Does a list container serve us better than a queue?

Reply:
It makes it easier (less code) when we populate with the filtered contents from the set.